.github/workflows/rit.yml

```diff
@@ -0,0 +1,68 @@
+name: Rootstock Integration Tests
+
+on:
+  pull_request:
+    types: [opened, reopened]
+    branches:
+      - "**"
```

Comment:
it's really necessary to run the tests against each PR created independently from the destination?

Reply:
Addressed. It’s not strictly necessary to run tests on every PR separately from the destination branch. I’ve adjusted the workflow to focus on push events for the `master` branch and branches ending with `-rc`

Reply:
@rmoreliovlabs , I believe @bcodesido meant that it can be optimised a bit if that's being triggered automatically only for master / *-rc. For PRs created against other branches (aka feature branches), manual / on-demand triggering should be sufficient. This should make the trick, shouldn't it?:
```
  pull_request:
    types: [opened, reopened]
    branches:
      - "master"
      - "*-rc"
```